README.md

```diff
@@ -165,14 +165,15 @@ It can be a Slack webhook or a custom API.
 [notifications]
   enabled = true
   [notifications.attacks]
+    schedule = "{\"text\": \"schedule: {$schedule}\"}"
```

Comment:
I think this should be separate from the attacks.
Maybe something like this:
```
[notifications]
  enabled = true
  [notifications.attacks]
   (…)
  [notifications.schedules]
   enabled = true
```

or like this:
```
[notifications]
  enabled = true
  reportSchedule = true
  [notifications.attacks]
   (…)
```

The separation would also make it simpler to, in a future pull request, have some control over the content of the schedule message (ie, we could use different placeholders like {$killTime}, {$victimName}, etc).

Reply:
Hello @worldtiki . Sure no problem. I will do as you suggested